app/database/queries/breeze_buddy/call_execution_config.py

```
@@ -222,88 +240,92 @@ def update_call_execution_config_query(
     Returns:
         Tuple of (query string, empty values list)
     """
     query = f"""
-        SELECT DISTINCT shop_identifier
+        SELECT DISTINCT COALESCE(merchant_identifier, shop_identifier) AS merchant_identifier
         FROM {CALL_EXECUTION_CONFIG_TABLE}
-        WHERE shop_identifier IS NOT NULL
-        ORDER BY shop_identifier ASC
+        WHERE COALESCE(merchant_identifier, shop_identifier) IS NOT NULL
+        ORDER BY merchant_identifier ASC
     """
```

Ruff (0.14.14)

[error] Possible SQL injection vector through string-based query construction

(S608)